<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Validators</class>
 <widget class="QMainWindow" name="Validators">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lab_alphanum">
       <property name="text">
        <string>Alphanumeric</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="le_alphanum"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lab_letters_space">
       <property name="text">
        <string>Lower case letters and spaces</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_lcspace"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lab_nums_1_100">
       <property name="text">
        <string>Numbers 1 to 100</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_nums_1_100"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lab_float">
       <property name="text">
        <string>Floating point</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="le_float"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="bt_save">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>386</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>Validators</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
